hetzner/README-cloud.md

```diff
@@ -91,6 +96,29 @@ It's used so Ansible can connect to the server without requiring password.
 
 # The VM
 
+## All-in-one creation
+
+2 new playbooks have been created to aggregate the different operations executed by the different playbooks
```

Comment:
Can you detail what the aggregate is supposed to do then ;-)

Reply:
Improved the docs to describe which playbooks the aggregate playbooks actually call